src/async_private.h

```diff
@@ -77,7 +78,7 @@ static inline void refreshTimeout(valkeyAsyncContext *ctx) {
     }
 }
 
-void valkeyAsyncDisconnectInternal(valkeyAsyncContext *ac);
-void valkeyProcessCallbacks(valkeyAsyncContext *ac);
+LIBVALKEY_API void valkeyAsyncDisconnectInternal(valkeyAsyncContext *ac);
+LIBVALKEY_API void valkeyProcessCallbacks(valkeyAsyncContext *ac);
```

Comment:
Why are these visible?

Reply:
These two, and `valkeySetError` are internal functions, but since `tls.c` and `libvalkey_tls.so` needs them I had to make them public. Maybe I should add a comment about it?

Reply:
Ah, OK. Yeah why not.